src/store/worker-profile/actions.js

```diff
@@ -0,0 +1,169 @@
+  let services = [];
+  let i = 0;
+  if (data.services)
+    data.services.forEach((element, index) => {
+      i++;
+      services.push(new VerificationRequestWorkerServiceModel({
+        name: element.name,
+        category: 1,//new ServiceCategoryModel({ id: String(element.category), name: "", code: String(element.category) }),
+        description: "23",
+        fee: String(element.fee),
+        minFee: element.minFee,
+        attachments: null //getAttachmentsByServiceIndex(serviceAttachments, index)
+      }))
+    })
+  return services;
+}
+
+const getEmploymentsArrayModel = (data) => {
+  let experiences = [];
+  if (data.experiences)
+    data.experiences.forEach((element) => {
+      experiences.push(new VerificationRequestWorkerEmploymentModel({
+        organization: element.organization,
+        since: element.since,
+        until: element.until,
+        responsibilities: element.responsibilities
+      }))
+    });
+  return experiences;
+}
+
+const workerProfileModelFromForm = (data, serviceAttachments) => {
+  console.log(data);
+  const currencies = getCurrenciesArrayModel(data);
+  const pageBackground = null;
+  const attachments = getAttachmentsArrayModel(data);
+  const socials = getSocialsArrayModel(data);
+  const services = getServicesArrayModel(data, serviceAttachments);
+  const employments = getEmploymentsArrayModel(data);
+  return new VerificationRequestWorkerModel({
+    regular: {
+      currencies: currencies,
+      hourlyCharge: data.hourlyCharge
+  },
+  verifiable: {
+      intro: data.intro,
+      pageBackground: pageBackground,
+      attachments: attachments
+  },
+  custom: null,
+  socials: socials,
+  services: services,
+  employments: employments
+  })
+}
+
+export const WORKER_PROFILE_REVIEW_REQUEST = 'WORKER_PROFILE/PROFILE_REVIEW/REQUEST'
+export const WORKER_PROFILE_REVIEW_SUCCESS = 'WORKER_PROFILE/PROFILE_REVIEW/SUCCESS'
+export const WORKER_PROFILE_REVIEW_FAILURE = 'WORKER_PROFILE/PROFILE_REVIEW/FAILURE'
+export const reviewWorkerProfileRequest = (req) => ({ type: WORKER_PROFILE_REVIEW_REQUEST, payload: req })
+export const reviewWorkerProfileSuccess = (res) => ({ type: WORKER_PROFILE_REVIEW_SUCCESS, payload: res })
+export const reviewWorkerProfileFailure = (err) => ({ type: WORKER_PROFILE_REVIEW_FAILURE, payload: err })
+export const reviewWorkerProfile = (data, serviceAttachments) => async (dispatch, getState) => {
+  const workerProfile = workerProfileModelFromForm(data, serviceAttachments);
+  try {
+    dispatch(reviewWorkerProfileRequest())
+    const state = getState()
+    const token = userTokenSelector()(state)
+    const res = await backendApi.submitWorkerProfile(workerProfile, token)
+    const initialValues = getWorkerProfileInitialValues(res.profile)
+    dispatch(initialize(WORKER_PROFILE_FORM, initialValues))
+    dispatch(reviewWorkerProfileSuccess(res.profile))
+  } catch (err) {
+    dispatch(reviewWorkerProfileFailure(err))
+  }
+}
```

Comment:
this action calls backendApi.submitWorkerProfile but name is reviewWorkerProfile
this action should named submitWorkerProfile